### PR TITLE
Fix bug with importCommand

### LINKED ISSUE #156: [PE-D][Tester D] add command allows duplicate d/ prefix
Doing 
add n/example p/910 d/today 10am-2pm d/today 10am-2pm
still allows the contact to be successfully added

-------------
Labels: `severity.High` `type.FunctionalityBug`
original: erv-teo/ped#11

### LINKED ISSUE #170: [PE-D][Tester D] Not able to import data
More specific details can be added for the import command in UG
Saving crash.csv to my downloads folder, and doing `import i/Downloads/crash.csv` or `import i/~/Downloads/crash.csv` both result in a data loading error even though my data does not have empty lines 

While it may be a user issue, error message given does not give details on how to fix nor does the UG 

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: erv-teo/ped#13

### LINKED ISSUE #176: [PE-D][Tester E] `import` command seems to not work
I try import a csv with one new patient and all similar patient that is already in the app.
Output: Error
Expected: one patient added.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: vnnamng/ped#17

### PR DESCRIPTION
**Pull request by SampsonYe1999**
Fix the issue when importing a csv file containing pre-existing patients, the result box shows duplicate person warning, and the importing process is preempted.

Update UG to better align with the excepted behaviors.

fixes #176
fixes #170 
fixes #156

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -149,7 +157,8 @@ You can export existing patient data to a csv file stored in a specific location
 
 Format: `export`
 
-- Please create a directory named "RapidTracerData" and create a new csv file named "PatientData" in the directory.
+- Create a directory named `RapidTracerData` within the file where RapidTracer is launched.
+- Create a new csv file named `PatientData.csv` in the `RapidTracerData` directory.
```
Comment (minreiseah) [inline docs/UserGuide.md:161]: Is it possible for the application to automatically generate the directory if it doesn't exist?

File: src/main/java/seedu/address/logic/commands/person/ImportCommand.java [hunk 2]
```diff
@@ -121,6 +132,32 @@ public List<Map<String, String>> readFile() throws DataLoadingException {
...
+        String phoneStr = patientDetail.get("phone");
+        String addressStr = patientDetail.get("address");
+        String tagsStr = patientDetail.get("tags");
+
+        // Parse tags from string to set of Tag objects
+        Set<Tag> tags = Arrays.stream(tagsStr.split(";"))
+                .map(String::trim)
+                .filter(tag -> !tag.isEmpty())
+                .map(Tag::new)
+                .collect(Collectors.toSet());
```
Comment (minreiseah) [inline src/main/java/seedu/address/logic/commands/person/ImportCommand.java:151]: Very clean.

**Review by minreiseah — DISMISSED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (SampsonYe1999) [reply docs/UserGuide.md:161]: Actually based on the createDirectory() method in the exportCommand class, the directory and file will be auto generated if they don't exist previously.

Comment (SampsonYe1999) [reply docs/UserGuide.md]: have added one remark in the UG to notify the user that if they don't create the directory and file as stated, they will be auto generated.

**Review by minreiseah — APPROVED** [review]
LGTM

**Pull request merged**